packages/wrangler/src/__tests__/utils-collect.test.ts

```diff
@@ -0,0 +1,39 @@
+import { collect } from "../utils/collect";
+
+// a test suite that handles the collect function
+describe("collect", () => {
```

Comment:
I like to make it clear things are functions.

```suggestion
describe("collect()", () => {
```